redaxo/src/core/default.config.yml

```diff
@@ -106,3 +105,11 @@ accesskeys:
 editor: null
 editor_basepath: null
 theme: null
+blocked_extensions: [php, php3, php4, php5, php6, php7, php8, phar, pht, phtml, hh, pl, asp, aspx, cfm, jsp, jsf, bat, sh, cgi, htaccess, htpasswd]
+# example:
+#   allowed_mime_types:
+#       gif: [image/gif]
+#       jpg: [image/jpeg, image/pjpeg]
+allowed_mime_types: ~
+allowed_doctypes: [avif, bmp, css, doc, docx, eps, gif, gz, jpg, jpeg, mov, mp3, mp4, ogg, pdf, png, ppt, pptx, pps, ppsx, rar, rtf, svg, swf, tar, tif, tiff, txt, webp, wma, xls, xlsx, zip]
```

Comment:
Die config.yml wird ja automatisch erstellt. Dabei wird bis zur zweiten Ebene aufgeklappt, erst ab dritter Ebene wird der Inline-Style genutzt. Die Extensions bekämen also alle eine eigene Zeile.
Daher überlege ich, ob wir für die Keys hier doch noch eine Ebene drüber setzen. Also zum Beispiel alles in `mediapool:` packen.

Reply:
<img src="https://github.com/redaxo/redaxo/assets/330436/75e877f2-86f9-47d1-9ecb-64500dbf933d" width=200/>

Also so landet es dann nach dem Setup in der config.yml, oder auch wenn man unter System was ändert. Also immer, wenn die config.yml beschrieben wird.

Aber ich werde es jetzt trotzdem erstmal so mergen und wann anders nochmal überlegen, wie man damit umgeht.